app/build.gradle

```diff
@@ -24,30 +24,34 @@ android {
     }
 
     buildTypes {
+        buildTypes {
+            all {
+                proguardFiles getDefaultProguardFile('proguard-android.txt'), "proguard.cfg"
+            }
+            debug {
+                minifyEnabled false
+                debuggable true
+            }
+
+            release {
+                minifyEnabled true
+                debuggable false
+            }
 
-        debug {
-            minifyEnabled false
-            debuggable true
         }
 
-        release {
-            minifyEnabled false
-            debuggable false
+        lintOptions {
+            disable 'ExtraTranslation'
+            disable 'MissingTranslation'
+            abortOnError false
         }
 
     }
 
-    lintOptions {
-        disable 'ExtraTranslation'
-        disable 'MissingTranslation'
-        abortOnError false
-    }
-
-}
+    final SUPPORT_LIBRARY_VERSION = rootProject.ext.SUPPORT_LIBRARY_VERSION
 
-final SUPPORT_LIBRARY_VERSION = rootProject.ext.SUPPORT_LIBRARY_VERSION
-
-dependencies {
-    compile "com.android.support:support-v4:${SUPPORT_LIBRARY_VERSION}"
-    compile 'com.squareup.picasso:picasso:2.5.2'
-}
+    dependencies {
+        compile "com.android.support:support-v4:${SUPPORT_LIBRARY_VERSION}"
+        compile 'com.squareup.picasso:picasso:2.5.2'
+    }
```

Comment:
Overall, the spacing is a bit messed up around here due to the additional buildTypes tag above. This commit should be cleaner here since only a few lines are being added, yet there are over 30 lines being changed.